scripts/ci-entrypoint.sh

```diff
@@ -31,6 +31,7 @@ KUSTOMIZE="${REPO_ROOT}/hack/tools/bin/kustomize"
 make --directory="${REPO_ROOT}" "${KUBECTL##*/}" "${HELM##*/}" "${KIND##*/}" "${KUSTOMIZE##*/}"
 KIND_CLUSTER_NAME="${KIND_CLUSTER_NAME:-capz}"
 WORKER_MACHINE_COUNT="${WORKER_MACHINE_COUNT:-2}"
+EXTRA_NODES="${EXTRA_NODES:-0}"
```

Comment:
Could we encode some baseline here like "create at most 100 nodes at first, then incrementally until we hit WORKER_MACHINE_COUNT" instead of needing to define a separate variable?

Reply:
Because `WORKER_MACHINE_COUNT` is already embedded into many scripts + the cluster templates themselves it will be difficult to unwind its role as the static number that represents the set of worker nodes to initially build a machinedeployment with.

Reply:
yes